docs/gallery/howto/autogen/parallel_wf.py

```diff
@@ -0,0 +1,245 @@
+# ```((x + y) + (u + v))``` to compute ```x + y``` and ```u + v``` in parallel.
+# aiida-core already provides a ArithmeticAddCalculation CalcJob for performing
+# addition which we will use it for this example
+
+from aiida_workgraph import WorkGraph, task
+from aiida.calculations.arithmetic.add import ArithmeticAddCalculation
+from aiida.orm import Int, InstalledCode, load_computer, load_code, load_node
+from aiida.common.exceptions import NotExistent
+
+# The ArithmeticAddCalculation needs to know where bash is stored
+try:
+    code = load_code("add@localhost")  # The computer label can also be omitted here
+except NotExistent:
+    code = InstalledCode(
+        computer=load_computer("localhost"),
+        filepath_executable="/bin/bash",
+        label="add",
+        default_calc_job_plugin="core.arithmetic.add",
+    ).store()
+
+wg = WorkGraph("parallel")
+x, y, u, v = (1, 2, 3, 4)
+add_xy = wg.add_task(ArithmeticAddCalculation, x=x, y=y, code=code)
+add_xy.set({"metadata.options.sleep": 5})  # the CalcJob will sleep 5 seconds
+add_uv = wg.add_task(ArithmeticAddCalculation, x=u, y=v, code=code)
+add_uv.set({"metadata.options.sleep": 5})  # the CalcJob will sleep 5 seconds
+add_xyuv = wg.add_task(
+    ArithmeticAddCalculation,
+    x=add_xy.outputs["sum"],
+    y=add_uv.outputs["sum"],
+    code=code,
+)
+wg.submit(wait=True)
+
+# %%
+#  We look at the ctime (the time of creation when submitted/run) and the mtime (the time the task has been last modified which is when its state changes to finish).
+print("add_xy created at:", add_xy.ctime.time(), "finished at:", add_xy.mtime.time())
+print("add_uv created at:", add_uv.ctime.time(), "finished at:", add_uv.mtime.time())
+
+# %%
+# We can see that both CalcJob's have been created almost at the same time
+
+# %%
+# Comparison with a calcfunction
+# ------------------------------
+#
+
+
+@task.calcfunction()
+def add(x, y, sleep):
+    import time
+
+    time.sleep(sleep.value)
+    return x + y
+
+
+wg = WorkGraph("parallel")
+x, y, u, v = (1, 2, 3, 4)
+add_xy = wg.add_task(add, x=x, y=y, sleep=5)
+add_uv = wg.add_task(add, x=x, y=y, sleep=5)
+add_xyuv = wg.add_task(
+    add, x=add_xy.outputs["result"], y=add_uv.outputs["result"], sleep=0
+)
+
+wg.submit(wait=True)
+
+# %%
+# Printing timings
+
+print("add_xy created at", add_xy.ctime.time(), "finished at", add_xy.mtime.time())
+print("add_uv created at", add_uv.ctime.time(), "finished at", add_uv.mtime.time())
+
+# %%
+# We can see that the calcfunctions have been run with a 5 seconds delay
+
+
+# %%
+# Parallelizing WorkGraphs
+# ========================
+# We will parallelize a workgraph by two ways, one time we submit all workgraphs,
+# the other time we use the graph builder to submit once the whole workflow.
+
+
+# This is our initial WorkGraph we want to parallelize
+@task.graph_builder(
+    inputs=[{"name": "integer"}], outputs=[{"name": "sum", "from": "sum_task.result"}]
+)
+def add10_wg(integer):
```

Comment:
`add10` because it adds 10 to a number. `wg` is I think a bit general indicating the problem that I have when I want to refer a graph builder. It can be a function as defined here, a task when integrated into a WorkGraph, and it is actually a WorkGraph that is executed. Because the whole tutorial is about parallelizing workgraphs I added `wg` into the name, but we can also remove it.

Reply:
renamed to `add10` I think that is more consistent with the rest